src/main/java/parser/Parser.java

```diff
@@ -21,7 +17,7 @@
      *
      * <Demo>        ::= {do <Algorithm> on <Graph> from <Node> to <Node>}
      * <Graph>       ::= {graph {<Nodes>} {<Edges>}}
-     * <Node>        ::= <string>
+     * <Node>        ::= <string>(changes)
```

Comment:
not sure what happened here?

Reply:
typing mistakes, i will remove it later.

Reply:
can we remove it here on github?

Reply:
i don't think so, you can just make the change in your branch and push

Reply:
ok